Ball of Duty Server/Services/BoDService.cs

```diff
@@ -172,5 +178,22 @@ public GameDTO JoinGame(int clientPlayerId, int clientUdpPort, int clientTcpPort
                 Debug.WriteLine($"Player: {clientPlayerId} failed quitting game" /*: {gameId}."*/);
             }
         }
+
+        public PlayerDTO[] GetLeaderboard()
+        {
+            Player[] players = DataModelFacade.GetTopPlayers();
```

Comment:
why didnt you just let GetTopPlayers return IEnumerable<Player>
and then use LINQ to generate this methods return value